src/components/Forms/AddRoleForm/AddRoleFormContainer.tsx

```diff
@@ -18,6 +18,8 @@ import { useAuth } from "../../../contexts/userContext";
 import { IGetUserRole } from "../../../models/access/query";
 import { GET_USER_ROLES } from "../../../graphql/User/query";
 import { MODULE_CODES } from "../../../utils/access";
+import { useLocation } from "react-router-dom";
+import { FORM_ACTIONS } from "../constant";
 
 const getInitialValues = (controllerActionHash: IControllerAction | {}): IAddRole => {
```

Comment:
Why are we allowing `{}` as parameter?

Reply:
we are allowing {} as a type because permissions from which we have made the hash can be an empty object